Octokit.Tests/Clients/AuthorizationsClientTests.cs

```diff
@@ -37,6 +37,19 @@ public void GetsAListOfAuthorizations()
                     Arg.Is<Uri>(u => u.ToString() == "authorizations"),
                     Args.ApiOptions);
             }
+
+            [Fact]
+            public void RequestsCorrectUrlWithApiOptions()
+            {
+                var client = Substitute.For<IApiConnection>();
+                var authEndpoint = new AuthorizationsClient(client);
+                
+                authEndpoint.GetAll(ApiOptions.None);
+
+                client.Received().GetAll<Authorization>(
+                    Arg.Is<Uri>(u => u.ToString() == "authorizations"),
+                    Args.ApiOptions);
```

Comment:
I'm fully agree, will be fixed in the next commits. Now I unmarked items in PR todo list, so WIP :)